saas/models/saas_operator.py

```diff
@@ -73,7 +73,7 @@ def generate_db_name(self):
 
     def _get_mandatory_args(self, db):
         self.ensure_one()
-        master_url = self.master_url
+        master_url = self.env['ir.config_parameter'].get_param('web.base.url')
```

Comment:
The same is the same as server-to-server communication. Can we rename one of them to avoid confusion? E.g. ``link_to_master`` for Server-to-server value ?

Reply:
Ok